app/src/main/java/no/usn/mob3000/ui/screens/AdministratorDashboardScreen.kt

```diff
@@ -261,15 +260,19 @@ fun UserListItem(
                 if (user.isAdmin) {
                     Button(
                         onClick = { onDemoteUser(user.id) },
-                        colors = ButtonDefaults.buttonColors(DefaultButton)
-                    ) {
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = MaterialTheme.colorScheme.primary,
+                            contentColor = MaterialTheme.colorScheme.onPrimary
+                        )                       ) {
                         Text(stringResource(R.string.admin_dashboard_fragment_button_demote))
                     }
                 } else {
                     Button(
                         onClick = { onPromoteUser(user.id) },
-                        colors = ButtonDefaults.buttonColors(DefaultButton)
-                    ) {
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = MaterialTheme.colorScheme.primary,
+                            contentColor = MaterialTheme.colorScheme.onPrimary
+                        )                    ) {
```

Comment:
Indentation. And [see this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).